migrate/20231129_github_feature_flags.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:github_installation) do
+      add_column :feature_flags, :jsonb, null: false, default: "{}"
```

Comment:
GithubInstallation has a project. We may use `feature_flags` column from the project. `github_runner.installation.project.feature_flags["storage_policy"]`

Reply:
Sounds good. I had assumed runners share the same project, which seems is not the case. I will make the change.

Reply:
The underlying vm and IPv4 resources share the same internal project. But the GithubRunner resources are in the customer project.